include/ff/parallel_execution_ff.h

```diff
@@ -62,18 +63,24 @@ class parallel_execution_ff {
   and ordering.
   \param concurrency_degree Number of threads used for parallel algorithms.
   \param order Whether ordered executions is enabled or disabled.
+  \note The concurrency_degree is a hint and the implementation might select
+        a lower value.
   */
   parallel_execution_ff(int concurrency_degree, bool order = true) noexcept :
-      concurrency_degree_{concurrency_degree},
+      concurrency_degree_{std::min(concurrency_degree, 
+          static_cast<int>(std::thread::hardware_concurrency()))},
```

Comment:
Does this mean that FF does not support oversubscription? Why?

Reply:
When increasing concurrency degree the error obtained is:

ERROR: FARM, try to add too many workers, please increase max_nworkers

Reply:
Ok.